### PR TITLE
Add managementGroups/{id}/descendants route to allow pagination of descendants

### PR DESCRIPTION
**Pull request by daxfohl**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/6245 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/6245/commits/f54798fcb4488bad377a704924dd6b2ce77d8a63

**Comment by daxfohl** [comment]
ARM docs say to do the review before updating manifest.  Here says functionality has to be available before review.  Everywhere says please make sure API is approved before implementing.  I'm confused.

**Comment by daxfohl** [comment]
Reviewing the documentation more, it seems like I should create a new api-version since it adds a new route.  Is this correct?

**Comment by jhendrixMSFT** [comment]
You only need to create a new API version if you're introducing breaking changes which doesn't appear to be the case here.
We cannot merge this PR until the API has been deployed, however we can review it here along with ARM's review.

File: specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json [hunk 1]
```diff
@@ -272,6 +272,44 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/providers/Microsoft.Management/managementGroups/{groupId}/descendants" : {
+      "post" : {
+        "tags" : [ "ManagementGroups_Descendants" ],
+        "operationId" : "ManagementGroups_GetDescendants",
+        "description" : "List all entities that descend from a management group.\n",
+        "parameters" : [ {
```
Comment (jhendrixMSFT) [inline specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json]: Nit, formatting (see previous operation ID).

Comment (daxfohl) [reply specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json]: Fixed formatting

**Comment by jhendrixMSFT** [comment]
@daxfohl is this API live or still waiting ARM review?

**Comment by daxfohl** [comment]
Still in ARM review.

**Comment by sanjaiganesh** [comment]
signed off after discussing with Gaurav.

**Comment by jhendrixMSFT** [comment]
@daxfohl is this new endpoint live?

**Comment by daxfohl** [comment]
It is, but someone noticed a case sensitivity bug.  So we should wait to
publish it until that's fixed later this week.

**Comment by jhendrixMSFT** [comment]
@daxfohl has this been resolved?

**Comment by daxfohl** [comment]
Yes, this can be merged now

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**